OpenEdXMobile/src/main/java/org/edx/mobile/base/MainApplication.java

```diff
@@ -133,24 +133,29 @@ private void init() {
     private void checkIfAppVersionUpgraded(Context context) {
         PrefManager.AppInfoPrefManager prefManager = new PrefManager.AppInfoPrefManager(context);
         long previousVersionCode = prefManager.getAppVersionCode();
-        final long currVersionCode = (long) BuildConfig.VERSION_CODE;
-        if (previousVersionCode < currVersionCode) {
+        final long curVersionCode = (long) BuildConfig.VERSION_CODE;
```

Comment:
Its good to cast it to long to make data type similar to `previousVersionCode` (which could only be long because of preferences storage) and use them in `AppUpdateEvent` class